Fix Workflow for releasing to PyPI

Currently the workflow fails with the following...

```
Run python setup.py sdist bdist_wheel
  python setup.py sdist bdist_wheel
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.9.13/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.13/x64/lib
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd[2](https://github.com/Mesnage-Org/pgfinder/runs/7359191998?check_suite_focus=true#step:6:2) [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help
error: invalid command 'bdist_wheel'
Error: Process completed with exit code 1.
```

And yet the same command run locally (i.e. `python setup.py sdist bdist_wheel`) builds the package fine.

This may be down to the wheel package not being installed in the Python environment (see [very old Python 2.7 issue](https://stackoverflow.com/questions/34819221/why-is-python-setup-py-saying-invalid-command-bdist-wheel-on-travis-ci) where the same error was encoutered under Travis CI).

The [wheel documentation](https://wheel.readthedocs.io/en/stable/user_guide.html) suggests adding the following to `setup.cfg` (since wheel is a `setuptools` extension)...

```
[bdist_wheel]
universal = 1
```

--- a/.github/workflows/pypi.yaml
+++ b/.github/workflows/pypi.yaml
@@ -10,16 +10,17 @@ jobs:
     runs-on: ubuntu-latest
     name: Publish package to PyPi
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - name: Setup Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           python-version: 3.9
       - name: Installing the package
         run: |
           pip3 install .
+          pip3 install .[pypi]
       - name: Build package
         run: |
           python setup.py sdist bdist_wheel

--- a/setup.cfg
+++ b/setup.cfg
@@ -48,8 +48,14 @@ docs =
   sphinxcontrib-napoleon
   sphinx-autodoc-typehints
 
+pypi =
+  wheel
+
 [options.packages.find]
 exclude = tests*
+
+[bdist_wheel]
+universal = 1
 
 [coverage:run]
 omit =

